recipes/quast/meta.yaml

```diff
@@ -1,19 +1,21 @@
 {% set name = "quast" %}
-{% set version = "4.6.0" %}
-{% set hash = "6ad4ef33b5f11e1d2efd3e15a285d0c8" %}
+{% set version = "4.6.1" %}
+{% set hash = "29118bff957af6d1b36136735450ddc8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 1
+  number: 0 
   string: boost{{CONDA_BOOST}}_{{PKG_BUILDNUM}}
 
 source:
   fn: {{ name|lower }}-{{ version }}.tar.gz
   url: http://downloads.sourceforge.net/project/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
   md5: {{ hash }}
+  patches:
+    - gnuplot_issue.patch
```

Comment:
Maybe use a filename that reflects the behavioral change, e.g., `disable-mummer-plots.patch`, or be more verbose, e.g., `gnuplot-issue_disable-mummer-plots.patch`.
To make the change in behavior more prominent, you can also add something along the lines of
```yaml
extra:
  notes: Drawing MUMmer plots is disabled due to issues with gnuplot.
```
at the end of `meta.yaml`.